akka-stream/src/main/scala/akka/stream/javadsl/Tcp.scala

```diff
@@ -281,7 +281,6 @@ class Tcp(system: ExtendedActorSystem) extends akka.actor.Extension {
       negotiateNewSession: NegotiateNewSession,
       backlog: Int,
       options: JIterable[SocketOption],
-      halfClose: Boolean,
```

Comment:
Even if it's ApiMayChange we shouldn't just change if we don't have a strong reason. Could be changed when we remove the ApiMayChange if `halfClose` was a mistake.

Reply:
I thought that the fact it wasn't used at all inside the method is readon enough?

Reply:
Hmm if it's not used then that is suspicious though... shouldn't it be used?

Reply:
Ah, so the inconsistency is that the underlying API has it, but the Scala API does not, yes, maybe it should rather be added there and passed along there.